README.md

```diff
@@ -88,17 +104,42 @@ You can check the application is running by calling the healthcheck endpoint wit
 curl localhost:{$PORT}/admin/healthcheck
 ```
 
-## Usage
+### `check`
 
-GOV.UK Verify provides prebuilt clients for the following languages and frameworks:
+You can run the VSP with the `check` command to confirm that your configuration file is valid. For example, to check that `verify-service-provider.yml` is valid, run:
+
+```
+./bin/verify-service-provider check verify-service-provider.yml
+```
+
+## Configure
+
+The VSP comes with a default [YAML configuration file](https://github.com/alphagov/verify-service-provider/blob/master/verify-service-provider.yml)
+called `verify-service-provider.yml` which you can customise either by providing environment variables or by editing the file directly.
+
+By default the following environment variables are supported:
+
+| Variable | Description |
+| -------- | ----------- |
+| `VERIFY_ENVIRONMENT` | The GOV.UK Verify Hub environment to run in.<br/>For example `PRODUCTION`, `INTEGRATION`|
+| `SERVICE_ENTITY_IDS` | A JSON string array with the service's entity ID, for example `'["http://entity-id"]'`. If you have several services using one VSP deployment,<br/> the array should contain all of their service entity IDs. |
+| `SAML_SIGNING_KEY` | A base64 encoded RSA private key used for signing the request to GOV.UK Verify Hub.|
+| `SAML_PRIMARY_ENCRYPTION_KEY`| A primary base64 encoded PKCS8 RSA private key used to decrypt SAML responses.|
+|`SAML_SECONDARY_ENCRYPTION_KEY`|(Optional - default empty) A secondary base64 encoded PKCS8 RSA private key is used to decrypt SAML responses. This parameter applies during [key rotation][key-rotation] events.|
+| `PORT` | (Optional - default `50400`) The TCP port where the application will listen for HTTP traffic|
+| `LOG_LEVEL` | Optional - default `INFO`) The threshold level for logs to be written, for example `DEBUG`, `INFO`, `WARN`, or `ERROR`) |
+
+If you are using the legacy version involving a [Matching Service Adapter (MSA)](https://github.com/alphagov/verify-matching-service-adapter), two additional environment variables apply:
+
+| Variable         | Description                         |
+| ---------------- | ----------------------------------- |
+| MSA_ENTITY_ID    | The `entityId` of the service's MSA |
```

Comment:
They probably need to edit the YAML file too so those environment variables can be used.

Reply:
This comment still applies. I think we should remove this table and leave the sentence on line 142, but remove the words: environment variables.

Reply:
Fair enough. I'll remove the table since they won't be able to use the information in the table anyway.  I've pushed a commit with this change 👌